### PR TITLE
[Frontend] tags with percentage

### PR DESCRIPTION
**Pull request by dyadyaJora**
implement block with language percentage
example
image

### CODE REVIEW

**Review by tcibinan — APPROVED** [review]
Looks nice! We should think about the tag colors later.

File: frontend/src/components/ActivityCard/ActivityCard.js [hunk 1]
```diff
@@ -1,22 +1,45 @@
...
+      contentInfo = (<div>
+        <p>Name: { name || '(not set)'}</p> 
+        <div>
+          { tags && Object.keys(tags).length ?
+              sortedTags.map((key, i) => 
+                <TagChip
+                  key={i}
+                  tag={key}
+                  real={Math.floor(tags[key]*100)}
+                  value={Math.floor(tags[key]*100) ? Math.floor(tags[key]*100) : '>1'}
```
Comment (tcibinan) [inline frontend/src/components/ActivityCard/ActivityCard.js]: Should be `<1` if I understand it correctly.

File: frontend/src/components/ActivityCard/ActivityCard.js [hunk 2]
```diff
@@ -1,22 +1,45 @@
...
+            return -1;
+          }
+        });
+      }
+
+      contentInfo = (<div>
+        <p>Name: { name || '(not set)'}</p> 
+        <div>
+          { tags && Object.keys(tags).length ?
+              sortedTags.map((key, i) =>
```
Comment (tcibinan) [inline frontend/src/components/ActivityCard/ActivityCard.js]: Probably, we can group tags that have weights lower than `0.01` into a single `Others` tag.

**Review by tcibinan — CHANGES_REQUESTED** [review]

File: frontend/src/components/ActivityCard/ActivityCard.js (hunk 2 above)
Comment (dyadyaJora) [reply frontend/src/components/ActivityCard/ActivityCard.js]: single tag others added

File: frontend/src/components/ActivityCard/ActivityCard.js (hunk 1 above)
Comment (dyadyaJora) [reply frontend/src/components/ActivityCard/ActivityCard.js]: ++

**Review by lerushe — APPROVED** [review]

**Review by tcibinan — APPROVED** [review]

**Pull request merged**